app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -31,18 +31,21 @@
 
   @include unselectable;
 
-  // 100dvh with fallback for old browsers
-  --100dvh: 100vh;
-  @supports (height: 100dvh) {
-    --100dvh: 100dvh;
+  // 1dvh with fallback for old browsers
+  --1dvh: 1vh;
+  @supports (height: 1dvh) {
+    --1dvh: 1dvh;
   }
 
-  height: calc(var(--100dvh) - var(--header-offset, 0px));
+  height: calc(
+    var(--composer-vh, var(--1dvh)) * 100 - var(--header-offset, 0px)
+  );
 
   .footer-nav-ipad & {
     top: calc(var(--header-offset) + var(--footer-nav-height));
     height: calc(
-      var(--100dvh) - var(--header-offset, 0px) - var(--footer-nav-height, 0px)
+      var(--composer-vh, var(--1dvh)) * 100 - var(--header-offset, 0px) -
+        var(--footer-nav-height, 0px)
```

Comment:
@pmusaraj do you know why we always define `--footer-nav-height` even when not on ipad? would be much more simple to be able to have it in regular code and just fallback to 0 when not defined

Reply:
@jjaffeux I believe it's because the footer nav is used in DiscourseHub (both mobile and iPad) as well as in PWAs (again, both mobile and iPad).

Reply:
Not sure which comment you're referring, I was referring to this: 

https://github.com/discourse/discourse/blob/main/app/assets/stylesheets/common/components/footer-nav.scss#L1-L10

Reply:
Well my bad then, I was quite certain to have read ipad here, cool then, thanks